src/extension.ts

```diff
@@ -124,18 +126,26 @@ export function activate(context: vscode.ExtensionContext): void {
     context.subscriptions.push(vscode.commands.registerCommand(
         `${SETTINGS_VIEW_NAME}.configureLaunchJson`,
         () => {
+            telemetryReporter.sendTelemetryEvent('user/buttonPress', {
+                'VSCode.buttonCode': launchConfig === 'None' ? buttonCode.generateLaunchJson : buttonCode.configureLaunchJson,
+            });
             void configureLaunchJson();
             launchConfig = getLaunchJson();
         }));
     context.subscriptions.push(vscode.commands.registerCommand(
         `${SETTINGS_VIEW_NAME}.launchProject`,
         () => {
+            telemetryReporter.sendTelemetryEvent('user/buttonPress', { 'VSCode.buttonCode': buttonCode.launchProject });
             launchConfig = getLaunchJson();
-            if (vscode.workspace.workspaceFolders && launchConfig) {
+            if (vscode.workspace.workspaceFolders && typeof launchConfig === 'object') {
```

Comment:
Let's set `typeof launchConfig === 'object'` to a const with a meaningful name to improve readability.  A suggestion would be something like this:
`const isValidLaunchConfig = typeof launchConfig === 'object';`